ultraLatency.py

```diff
@@ -64,7 +61,7 @@
     step = options.scanmax - options.scanmin
 
 if options.debug:
-    uhal.setLogLevelTo(uhal.LogLevel.INFO)
+    uhal.setLogLevelTo(uhal.LogLevel.DEBUG)
```

Comment:
there is pretty much no case where when debugging a tool i'm interested in seeing the `uhal::LogLevel::DEBUG` messages, `uhal::LogLevel::INFO` should be sufficient